FoodOrderWebApi/Configuration/DbInitializer.cs

```diff
@@ -0,0 +1,215 @@
+{
+    public class DbInitializer
+    {
+        private static FoodOrderDbContext _context = null!;
+
+        public static void Initialize(IServiceProvider serviceProvider)
+        {
+            _context = serviceProvider.GetRequiredService<FoodOrderDbContext>();
+
+            if (!_context.Database.EnsureCreated())
+            {
+                return;
+            }
+
+            SeedCategories();
+            SeedOpeningHours();
+            SeedRestaurants();
+            SeedProducts();
+        }
+        private static void SeedCategories()
+        {
+            var categories = new Category[]
+            {
+                new Category
+                {
+                    Name = "Soup",
+                    PictureName = "soup.jpg"
+                },
+                new Category
+                {
+                    Name = "Drink",
+                    PictureName = "drink.jpg"
+                },
+                new Category
+                {
+                    Name = "Pizza",
+                    PictureName = "pizza.jpg"
+                },
+                new Category
+                {
+                    Name = "Pasta",
+                    PictureName = "pasta.jpg"
+                },
+            };
+
+            foreach (Category c in categories)
+            {
+                _context.Categories.Add(c);
+            }
+
+            _context.SaveChanges();
+        }
+
+        private static void SeedRestaurants()
+        {
+            var restaurants = new Restaurant[]
+            {
+                new Restaurant
+                {
+                    Name = "Karen bar",
+                    Description = "something",
+                    Address = "Budapest, Lágymányosi campus",
+                    PhoneNumber = "06300000001",
+                    LogoName = "karen_bar.jpg",
+                    OpeningHourId = 1,
+                    ClosingHourId = 2,
+                },
+                new Restaurant
+                {
+                    Name = "Blue Parrot",
+                    Description = "soemthing",
+                    Address = "Budapest, Wesselényi street",
+                    PhoneNumber = "06300000002",
+                    LogoName = "blue_parrot.jpg",
+                    OpeningHourId = 3,
+                    ClosingHourId = 4,
+                },
+            };
+
+            foreach (Restaurant r in restaurants)
+            {
+                _context.Restaurants.Add(r);
+            }
+
+            _context.SaveChanges();
+        }
+
+        private static void SeedProducts()
+        {
+            var products = new Product[]
+            {
+                new Product
+                {
+                    Name = "Goulash Soup",
+                    Description = "0,5L soup",
+                    Price= 1200,
+                    PictureName = "",
+                    CategoryName = "Soup",
+                    RestaurantId = 1,
+                },
```

Comment:
You could try to make a factory method for this as well, though it might not improve readability in this case